rocky/katalogus/views/plugin_enable_disable.py

```diff
@@ -91,5 +92,5 @@ def post(self, request, *args, **kwargs):
                     self.plugin.scan_level.value,
                 ),
             )
-
-        return HttpResponseRedirect(request.POST.get("current_url"))
+            url = request.build_absolute_uri(request.POST.get("current_url"))
+            return HttpResponseRedirect(url)
```

Comment:
The indentation changed here which results in the unit test errors.

The change also doesn't fix anything, because if you give an absolute uri to `request.build_absolute_uri` it will return the same uri.

Reply:
Nice catch.

What would be the best solution in this case? Check against an allowed domain list?

Reply:
You are right, its a False positive in combination with our use of CSRF. There's no way to influence the current_url param and still make it trough the CSRF without the user being involved. This PR can be closed.
I also did not notice the indentation change. Weird that it happend in the first place , but probably because of my copying of the line from the earlier method.